style.css

```diff
@@ -0,0 +1,100 @@
+@import url('https://fonts.googleapis.com/css2?family=Playfair+Display&family=Poppins&display=swap');
+
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+body {
+  width: 400px;
+  margin: auto;
+  height: 4910px;
+  position: relative;
+  background: #fff;
+}
```

Comment:
[OPTIONAL] Please do not use fixed width and height on body as we need our page to be responsive, do not implemets the same styles properties you see in figma. Mastering flex is a crucial skills for front-end development please use [Flexbox](https://css-tricks.com/snippets/css/a-guide-to-flexbox/) as much as possible and become a flex ninja :ninja: